<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>838</width>
    <height>762</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <property name="styleSheet">
   <string notr="true"/>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QVBoxLayout" name="verticalLayout_9">
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_5" stretch="3,5">
      <item>
       <layout class="QVBoxLayout" name="verticalLayout_7" stretch="8,1">
        <item>
         <widget class="QTextBrowser" name="textBrowser">
          <property name="styleSheet">
           <string notr="true">font: 14pt &quot;Arial&quot;;
color: rgb(0, 255, 0);
background-color: rgb(0, 0, 0);</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="Btn_clear">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Minimum" vsizetype="Preferred">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="font">
           <font>
            <pointsize>19</pointsize>
           </font>
          </property>
          <property name="text">
           <string>清空</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QVBoxLayout" name="verticalLayout_8" stretch="7,1">
        <item>
         <layout class="QVBoxLayout" name="verticalLayout_6" stretch="2,3,9,3">
          <item>
           <widget class="QLabel" name="label_3">
            <property name="font">
             <font>
              <family>Arial</family>
              <pointsize>34</pointsize>
              <weight>10</weight>
              <italic>false</italic>
              <bold>false</bold>
             </font>
            </property>
            <property name="styleSheet">
             <string notr="true">background-color: rgb(25, 85, 255);
color: rgb(255, 255, 255);
font: 87 34pt &quot;Arial&quot;;</string>
            </property>
            <property name="text">
             <string>Windeply</string>
            </property>
            <property name="alignment">
             <set>Qt::AlignCenter</set>
            </property>
           </widget>
          </item>
          <item>
           <layout class="QHBoxLayout" name="horizontalLayout_2">
            <item>
             <layout class="QHBoxLayout" name="horizontalLayout">
              <item>
               <layout class="QVBoxLayout" name="verticalLayout_2">
                <item>
                 <widget class="QLabel" name="label_2">
                  <property name="font">
                   <font>
                    <pointsize>12</pointsize>
                   </font>
                  </property>
                  <property name="text">
                   <string>AppPath:</string>
                  </property>
                 </widget>
                </item>
                <item>
                 <widget class="QLabel" name="label">
                  <property name="font">
                   <font>
                    <pointsize>12</pointsize>
                   </font>
                  </property>
                  <property name="text">
                   <string>qmldir:</string>
                  </property>
                 </widget>
                </item>
                <item>
                 <widget class="QLabel" name="label_5">
                  <property name="text">
                   <string>Qt_Dir:</string>
                  </property>
                 </widget>
                </item>
               </layout>
              </item>
              <item>
               <layout class="QVBoxLayout" name="verticalLayout_3">
                <item>
                 <widget class="QLineEdit" name="AppPath">
                  <property name="sizePolicy">
                   <sizepolicy hsizetype="Expanding" vsizetype="Preferred">
                    <horstretch>0</horstretch>
                    <verstretch>0</verstretch>
                   </sizepolicy>
                  </property>
                  <property name="font">
                   <font>
                    <pointsize>14</pointsize>
                   </font>
                  </property>
                  <property name="styleSheet">
                   <string notr="true">background-color: rgb(203, 209, 205);
color: rgb(0, 0, 0);</string>
                  </property>
                  <property name="readOnly">
                   <bool>true</bool>
                  </property>
                 </widget>
                </item>
                <item>
                 <widget class="QLineEdit" name="qmldir">
                  <property name="sizePolicy">
                   <sizepolicy hsizetype="Expanding" vsizetype="Preferred">
                    <horstretch>0</horstretch>
                    <verstretch>0</verstretch>
                   </sizepolicy>
                  </property>
                  <property name="font">
                   <font>
                    <pointsize>14</pointsize>
                   </font>
                  </property>
                 </widget>
                </item>
                <item>
                 <widget class="QLineEdit" name="pathC">
                  <property name="sizePolicy">
                   <sizepolicy hsizetype="Expanding" vsizetype="Preferred">
                    <horstretch>0</horstretch>
                    <verstretch>0</verstretch>
                   </sizepolicy>
                  </property>
                 </widget>
                </item>
               </layout>
              </item>
             </layout>
            </item>
            <item>
             <layout class="QVBoxLayout" name="verticalLayout_4">
              <item>
               <widget class="QToolButton" name="toolButton">
                <property name="sizePolicy">
                 <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
                  <horstretch>0</horstretch>
                  <verstretch>0</verstretch>
                 </sizepolicy>
                </property>
                <property name="text">
                 <string>...</string>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QToolButton" name="TB_qmldir">
                <property name="sizePolicy">
                 <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
                  <horstretch>0</horstretch>
                  <verstretch>0</verstretch>
                 </sizepolicy>
                </property>
                <property name="text">
                 <string>...</string>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QToolButton" name="toolButton_2">
                <property name="sizePolicy">
                 <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
                  <horstretch>0</horstretch>
                  <verstretch>0</verstretch>
                 </sizepolicy>
                </property>
                <property name="text">
                 <string>...</string>
                </property>
               </widget>
              </item>
             </layout>
            </item>
           </layout>
          </item>
          <item>
           <layout class="QHBoxLayout" name="horizontalLayout_3" stretch="1,0">
            <property name="spacing">
             <number>0</number>
            </property>
            <item>
             <layout class="QVBoxLayout" name="verticalLayout_5" stretch="0,10">
              <property name="spacing">
               <number>0</number>
              </property>
              <item>
               <widget class="QLabel" name="label_4">
                <property name="font">
                 <font>
                  <pointsize>18</pointsize>
                 </font>
                </property>
                <property name="styleSheet">
                 <string notr="true">color: rgb(255, 255, 255);
background-color: rgb(0, 0, 0);</string>
                </property>
                <property name="text">
                 <string>AppName</string>
                </property>
                <property name="alignment">
                 <set>Qt::AlignCenter</set>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QLabel" name="appName">
                <property name="font">
                 <font>
                  <family>SF Compact Display</family>
                  <pointsize>29</pointsize>
                  <weight>9</weight>
                  <italic>false</italic>
                  <bold>false</bold>
                 </font>
                </property>
                <property name="layoutDirection">
                 <enum>Qt::LeftToRight</enum>
                </property>
                <property name="styleSheet">
                 <string notr="true">font: 75 29pt &quot;SF Compact Display&quot;;
background-color: rgb(0, 0, 0);
color: rgb(0, 255, 0);</string>
                </property>
                <property name="text">
                 <string>Not Select App</string>
                </property>
                <property name="alignment">
                 <set>Qt::AlignCenter</set>
                </property>
               </widget>
              </item>
             </layout>
            </item>
            <item>
             <layout class="QVBoxLayout" name="verticalLayout">
              <property name="spacing">
               <number>0</number>
              </property>
              <item>
               <widget class="QCheckBox" name="CB_QML">
                <property name="sizePolicy">
                 <sizepolicy hsizetype="Minimum" vsizetype="Preferred">
                  <horstretch>0</horstretch>
                  <verstretch>0</verstretch>
                 </sizepolicy>
                </property>
                <property name="font">
                 <font>
                  <pointsize>14</pointsize>
                 </font>
                </property>
                <property name="styleSheet">
                 <string notr="true">background-color: rgb(0, 0, 0);
color: rgb(255, 255, 255);</string>
                </property>
                <property name="text">
                 <string>QML-Dir</string>
                </property>
                <property name="checked">
                 <bool>true</bool>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QCheckBox" name="CB_WebKit">
                <property name="sizePolicy">
                 <sizepolicy hsizetype="Minimum" vsizetype="Preferred">
                  <horstretch>0</horstretch>
                  <verstretch>0</verstretch>
                 </sizepolicy>
                </property>
                <property name="font">
                 <font>
                  <pointsize>14</pointsize>
                 </font>
                </property>
                <property name="styleSheet">
                 <string notr="true">background-color: rgb(0, 0, 0);
color: rgb(255, 255, 255);</string>
                </property>
                <property name="text">
                 <string>NO-WebKit</string>
                </property>
                <property name="checked">
                 <bool>true</bool>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QCheckBox" name="CB_Runtime">
                <property name="sizePolicy">
                 <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
                  <horstretch>0</horstretch>
                  <verstretch>0</verstretch>
                 </sizepolicy>
                </property>
                <property name="font">
                 <font>
                  <pointsize>14</pointsize>
                 </font>
                </property>
                <property name="styleSheet">
                 <string notr="true">background-color: rgb(0, 0, 0);
color: rgb(255, 255, 255);</string>
                </property>
                <property name="text">
                 <string>NO-Runtime</string>
                </property>
                <property name="checked">
                 <bool>true</bool>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QCheckBox" name="CB_Release">
                <property name="sizePolicy">
                 <sizepolicy hsizetype="Expanding" vsizetype="Preferred">
                  <horstretch>0</horstretch>
                  <verstretch>0</verstretch>
                 </sizepolicy>
                </property>
                <property name="font">
                 <font>
                  <pointsize>14</pointsize>
                 </font>
                </property>
                <property name="styleSheet">
                 <string notr="true">background-color: rgb(0, 0, 0);
color: rgb(255, 255, 255);</string>
                </property>
                <property name="text">
                 <string>Release</string>
                </property>
                <property name="checked">
                 <bool>true</bool>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QCheckBox" name="CB_3D">
                <property name="sizePolicy">
                 <sizepolicy hsizetype="Expanding" vsizetype="Preferred">
                  <horstretch>0</horstretch>
                  <verstretch>0</verstretch>
                 </sizepolicy>
                </property>
                <property name="font">
                 <font>
                  <pointsize>14</pointsize>
                 </font>
                </property>
                <property name="styleSheet">
                 <string notr="true">background-color: rgb(0, 0, 0);
color: rgb(255, 255, 255);</string>
                </property>
                <property name="text">
                 <string>NO-3D</string>
                </property>
                <property name="checked">
                 <bool>true</bool>
                </property>
               </widget>
              </item>
             </layout>
            </item>
           </layout>
          </item>
          <item>
           <widget class="QPushButton" name="Run">
            <property name="sizePolicy">
             <sizepolicy hsizetype="Minimum" vsizetype="Expanding">
              <horstretch>0</horstretch>
              <verstretch>0</verstretch>
             </sizepolicy>
            </property>
            <property name="styleSheet">
             <string notr="true">font: 87 24pt &quot;Arial&quot;;</string>
            </property>
            <property name="text">
             <string>Run</string>
            </property>
            <property name="icon">
             <iconset resource="res.qrc">
              <normaloff>:/res/start.png</normaloff>
              <normalon>:/res/pause.png</normalon>
              <disabledoff>:/res/start-disable.png</disabledoff>
              <disabledon>:/res/start-disable.png</disabledon>:/res/start.png</iconset>
            </property>
            <property name="iconSize">
             <size>
              <width>48</width>
              <height>48</height>
             </size>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_4" stretch="1,1,1">
          <item>
           <widget class="QPushButton" name="OpenDir">
            <property name="enabled">
             <bool>true</bool>
            </property>
            <property name="sizePolicy">
             <sizepolicy hsizetype="Minimum" vsizetype="Preferred">
              <horstretch>0</horstretch>
              <verstretch>0</verstretch>
             </sizepolicy>
            </property>
            <property name="font">
             <font>
              <pointsize>14</pointsize>
             </font>
            </property>
            <property name="text">
             <string>打开生成</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="pushButton_3">
            <property name="sizePolicy">
             <sizepolicy hsizetype="Minimum" vsizetype="Preferred">
              <horstretch>0</horstretch>
              <verstretch>0</verstretch>
             </sizepolicy>
            </property>
            <property name="font">
             <font>
              <pointsize>14</pointsize>
             </font>
            </property>
            <property name="text">
             <string>Reserve</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="reserve_temp">
            <property name="sizePolicy">
             <sizepolicy hsizetype="Minimum" vsizetype="Preferred">
              <horstretch>0</horstretch>
              <verstretch>0</verstretch>
             </sizepolicy>
            </property>
            <property name="font">
             <font>
              <pointsize>14</pointsize>
             </font>
            </property>
            <property name="text">
             <string>Reserve</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
       </layout>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>838</width>
     <height>22</height>
    </rect>
   </property>
   <widget class="QMenu" name="menu">
    <property name="title">
     <string>帮助</string>
    </property>
    <addaction name="action"/>
    <addaction name="action_2"/>
    <addaction name="separator"/>
   </widget>
   <widget class="QMenu" name="menu_2">
    <property name="title">
     <string>设置</string>
    </property>
    <addaction name="actionQTDIR"/>
   </widget>
   <addaction name="menu"/>
   <addaction name="menu_2"/>
  </widget>
  <action name="action">
   <property name="text">
    <string>关于</string>
   </property>
  </action>
  <action name="action_2">
   <property name="text">
    <string>使用说明</string>
   </property>
  </action>
  <action name="actionQTDIR">
   <property name="text">
    <string>QTDIR</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources>
  <include location="res.qrc"/>
 </resources>
 <connections/>
</ui>
